cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -912,19 +914,216 @@ def create_image(self, name):
         raise NotImplementedError(
             'To be implemented after GCEVolumeService.')
 
+    @property
```

Comment:
I'm wondering about the choice to make this a property, rather than an explicit function `get_target_instance(self)`. It feels like the labeling this a property leads to confusion when reading the code. "get_target_instance" or "lookup_target_instance" makes it more clear that there will be API calls involved.